app/assets/stylesheets/pages/_index.scss

```diff
@@ -1,13 +1,6 @@
 // Import page-specific CSS files here.
 @import "home";
 
-.column {
```

Comment:
@EmmaSelvig  with this proposed change (in this pull request) we would not need this class as we could use a similar class already defined in bootstrap, see bellow in the part I link the example of album